ckanext/ytp/comments/util.py

```diff
@@ -27,3 +28,15 @@ def clean_input(comment):
             template = "An exception of type {0} occurred. Arguments:\n{1!r}"
             message = template.format(type(e).__name__, e.args)
             log.debug(message)
+
+def remove_HTML_markup(text):
+    try:
+        # Returns the text content of the element, including the text content of its children, with no HTML markup.
+        return fromstring(text.replace('<br/>', '\n')).text_content()
+    except Exception, e:
+        if type(e).__name__ == "ParserError":
+            raise logic.ValidationError("Text is required")
+        else:
+            template = "An exception of type {0} occurred. Arguments:\n{1!r}"
+            message = template.format(type(e).__name__, e.args)
+            log.debug(message)
```

Comment:
should we have a new line at the end of the file?

Reply:
@duttonw Added new line to end of the file